Sources/AblyChat/RoomLifecycleManager.swift

```diff
@@ -752,9 +810,16 @@ internal actor DefaultRoomLifecycleManager<Contributor: RoomLifecycleContributor
                 case .suspended:
                     // CHA-RL1h2
                     let error = ARTErrorInfo(chatError: .attachmentFailed(feature: contributor.feature, underlyingError: contributorAttachError))
-                    changeStatus(to: .suspendedAwaitingStartOfRetryOperation(error: error))
 
                     // CHA-RL1h3
+                    // My understanding is that, since this task is being created inside an actor’s synchronous code, the two .suspended* statuses will always come in the right order; i.e. first .suspendedAwaitingStartOfRetryOperation and then .suspended.
+                    let retryOperationTask = scheduleAnOperation(
```

Comment:
The task is immediately being run here, shouldn't there be a timeout?

Reply:
What do you mean by "immediately run"? And why should there be a timeout? The spec just says "The room lifecycle shall asynchronously (non-blocking to the @ATTACH@ operation) enter the recovery loop"; isn't that what's being done here?

Reply:
Ok, there is no delay in the spec. Resolving then.